### PR TITLE
Updates to v2 API for functional tests

### PR DESCRIPTION
**Pull request by brianparry**
* Added content parameter to some /pollers routes
* Remove pollers pause and resume routes

### CODE REVIEW

**Comment by brianparry** [comment]
@RackHD/corecommitters @srinia6 @geoff-reid @abildw @dalebremner

**Comment by brianparry** [comment]
test this please

**Comment by abildw** [comment]
+1

File: lib/services/pollers-api-service.js [hunk 1]
```diff
@@ -9,13 +9,15 @@ di.annotate(pollerApiServiceFactory, new di.Provide('Http.Services.Api.Pollers')
...
+        'Protocol.Task',
+        '_'
     )
 );
 
 function pollerApiServiceFactory(
     waterline,
-    taskProtocol
+    taskProtocol,
+    _
```
Comment (geoff-reid) [inline lib/services/pollers-api-service.js]: Why are you removing lodash?  It is still used on line 84.

**Comment by geoff-reid** [comment]
+1 :+1:

**Comment by jlongever** [comment]
+1

**Pull request merged**